services/QuillLMS/lib/tasks/temporary/users.rake

```diff
@@ -90,4 +90,13 @@ namespace :users do
     User.where(id: user_ids).destroy_all
     puts "Deleted #{user_ids.count} users"
   end
+
+  task backfill_capitalize_names: :environment do
+    progress_bar = ProgressBar.new(User.count)
+
+    User.find_each do |user|
+      user.capitalize_name
```

Comment:
Do we need a `save` call after the `capitalize_name` call?

Reply:
Great point.  This actually doesn't do anything at the moment.